fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/logging/LogMessageKeys.java

```diff
@@ -253,38 +254,43 @@
     TIME_TO_WAIT_MILLIS("time_to_wait_milliseconds"),
 
     // Log the name of the tokenizer used
-    TOKENIZER_NAME("tokenizer_name"),
+    TOKENIZER_NAME,
 
     //for logging asyncToSync timeout limits
-    TIME_LIMIT("time_limit"),
-    TIME_UNIT("time_unit"),
+    TIME_LIMIT,
+    TIME_UNIT,
 
     // ranked set
-    HASH_FUNCTION("hash_function"),
-    HASH("hash"),
+    HASH_FUNCTION,
+    HASH,
 
     // Lucene data compression
-    ANALYZER_NAME("analyzer_name"),
-    BLOCK_NUMBER("block_number"),
-    COMPRESSED_EVENTUALLY("compressed_eventually"),
-    COMPRESSION_SUPPOSED("compression_supposed"),
-    COMPRESSION_VERSION("compression_version"),
-    DATA_SIZE("data_size"),
-    DATA_VALUE("data_value"),
-    ENCODED_DATA_SIZE("encoded_data_size"),
-    ENCRYPTED_EVENTUALLY("encrypted_eventually"),
-    ENCRYPTION_SUPPOSED("encryption_supposed"),
-    FILE_ID("file_id"),
-    FILE_NAME("file_name"),
-    FILE_REFERENCE("file_reference"),
-    ORIGINAL_DATA_SIZE("original_data-size"),
+    ANALYZER_NAME,
+    BLOCK_NUMBER,
+    COMPRESSED_EVENTUALLY,
+    COMPRESSION_SUPPOSED,
+    COMPRESSION_VERSION,
+    DATA_SIZE,
+    DATA_VALUE,
+    ENCODED_DATA_SIZE,
+    ENCRYPTED_EVENTUALLY,
+    ENCRYPTION_SUPPOSED,
+    FILE_ID,
+    FILE_NAME,
+    FILE_REFERENCE,
+    ORIGINAL_DATA_SIZE,
```

Comment:
There's an inconsistency here, where it uses a `-` instead of a `_`
```suggestion
    ORIGINAL_DATA_SIZE("original_data-size"),
```

Reply:
This change was deliberate, as noted in this comment: https://github.com/FoundationDB/fdb-record-layer/pull/1484#discussion_r764766278 The initial hyphen was (presumably) unintentional. I'm not actually sure what keys loggers generally accept, but I think it would generally be good to keep these as a-z and underscore